in_toto/verifylib.py

```diff
@@ -1423,6 +1432,12 @@ def in_toto_verify(layout, layout_key_dict, link_dir_path=".",
 
     step_name (optional): A name assigned to the returned link. This is mostly
         useful during recursive sublayout verification.
+    
+    persist_inspection_links: (optional)
+            A boolean that determines whether or not link metadata files for
+            inspection are written to cwd.
+            When true, the link inspection file is written to cwd.
+            When false, the file is not written.
```

Comment:
Minor nit: Can you un-indent one level to be consistent with the other arguments? Also, I think the the first two lines of the description should be enough to understand how to use the argument.